startup.sh

```diff
@@ -87,4 +98,6 @@ case $ARCH in
 		;;
 esac
 
-$JAVA_EXE -Dsun.java2d.dpiaware=false -Djava.library.path=${NATIVE_LIB_PATH} -classpath "$OPENHOSPITAL_CLASSPATH:$CLASSPATH" org.isf.menu.gui.Menu "$@"
+######### OPENHOSPITAL STARTUP
+
+cd $OPENHOSPITAL_HOME&& $JAVA_BIN -Dsun.java2d.dpiaware=false -Djava.library.path=${NATIVE_LIB_PATH} -classpath "$OPENHOSPITAL_CLASSPATH" org.isf.menu.gui.Menu "$@"
```

Comment:
```suggestion
pushd $OPENHOSPITAL_HOME
$JAVA_BIN -Dsun.java2d.dpiaware=false -Djava.library.path=${NATIVE_LIB_PATH} -classpath "$OPENHOSPITAL_CLASSPATH" org.isf.menu.gui.Menu "$@"
popd
```

Reply:
I agree with the change but we need to use the Bash shell to have pushd / popd.
Are we willing to use bash instead of the simple (and always available) /bin/sh ?

Reply:
good point. 
I'd say try `cd $OPENHOSPITAL_HOME` and `cd -` instead. 
Or save `$PWD` in a variable and restore it with `cd $myvar`.
Otherwise yes, I'd tend to believe bash or zsh are installed by default in most Linux desktop machines nowadays, so if the above doesn't work, I'd go with push/popd. Thanks.

Reply:
Agreed. Modified the script accordingly.